test/fhirtypes/StructureDefinition.test.ts

```diff
@@ -1083,11 +1083,69 @@ describe('StructureDefinition', () => {
       }).toThrow('Cannot resolve element from path: extension[fake-extension].value[x]');
     });
 
-    it('should allow fixing an InstanceDefinition to a resource element', () => {
-      const instanceDef = new InstanceDefinition();
-      instanceDef.resourceType = 'Patient';
-      const { fixedValue } = respRate.validateValueAtPath('contained[0]', instanceDef, fisher);
-      expect(fixedValue.resourceType).toBe('Patient');
+    describe('#Inline Instances', () => {
```

Comment:
Should there be an additional test for fixing a `Bundle`-type instance definition to a `Resource`? The tests right now show what `Bundle` can't be bound to, but not what it can be bound to.

Reply:
Yeah! I will add that.

Reply:
Added in https://github.com/FHIR/sushi/pull/359/commits/e35a0219dcfb9496d06c6a86ad73c392d460e863!